src/lib/Components/Bidding/Screens/ConfirmFirstTimeBid.tsx

```diff
@@ -31,32 +47,67 @@ export interface Address {
   postalCode: string
 }
 
-interface ConformBidState {
+interface StripeToken {
+  tokenId: string
+  created: number
+  livemode: 1 | 0
+  card: any
+  bankAccount: any
+  extra: any
+}
+
+interface ConfirmBidState {
   billingAddress?: Address
-  creditCardToken?: string // TODO: change this interface accrodingly when adapting stripe
+  creditCardFormParams?: PaymentCardTextFieldParams
+  creditCardToken?: StripeToken
   conditionsOfSaleChecked: boolean
   isLoading: boolean
 }
 
+const MAX_POLL_ATTEMPTS = 20
+
+const creditCardMutation = graphql`
+  mutation ConfirmFirstTimeBidMutation($input: CreditCardInput!) {
```

Comment:
How are we structuring these mutations? Are we creating the credit card right away after they exit the credit card screen? Or are we waiting until they click to confirm their bid to create the credit card (on gravity's side) + register/bid?

If the former, this naming is a little confusing since `ConfirmFirstTimeBid` suggests that it will mutate a `BidderPosition` object, when in reality this mutation is just creating a credit card. Maybe something like `CreateCreditCardMutation`?

Reply:
It's the latter, but the naming could be better for sure. Let's go with `CreateCreditCardMutation` to be more explicit.

Reply:
Sadly, Relay Modern wants this to always be `ConfirmFirstTimeBidMutation`:

```
Parse error: Error: RelayFindGraphQLTags: Operation names in graphql tags must be prefixed with the module name and end in "Mutation", "Query", or "Subscription". Got `CreateCreditCardMutation` in module `ConfirmFirstTimeBid`. in "lib/Components/Bidding/Screens/ConfirmFirstTimeBid.tsx"
```

Reply:
lol why? I trust there's no way around, just curious. 😅